neevaco/neeva-android
# Add support for users that cannot get free trial

* Changes the purchase subscription button text if Free Trial is not available to the user. 
* Renames the `getSubscriptionOfferDetails()` function to `getAllSubscriptionOfferDetails()`




## Checklists

### Pointers to help with review
* Hwi giving the string to use if Free Trial is not available: https://neevaco.slack.com/archives/C02RYG8LY8H/p1677642121950829

### How was this tested?

### Manual test cases
Took away the free trial for `premium-monthly` on Play Console. On a release build, you can see that the free-trial removal is rendered:

https://user-images.githubusercontent.com/20916043/222950756-696530fd-be18-475b-a811-3d09ffec8cc1.mp4
